CHANGELOG.md

```diff
@@ -1,7 +1,18 @@
 Change Log
 ==========
 
-Version 1.0.7 *(2019-10-01)*
+Version 1.0.9 *(2019-11-07)*
+----------------------------
+* Session Attributes - allow you to add session-specific information to a new message. [See docs for more information](https://developer.helpscout.com/beacon-2/android/#session-attributes)
```

Comment:
Session Attributes: You are now able to include session-specific information on new messages. [Check out our docs](https://developer.helpscout.com/beacon-2/android/#session-attributes) for more information.